### PR TITLE
fix(messages): Improve warn messages

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Improves some warn messages

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16968

### CODE REVIEW

File: src/tasks/installers/helm.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ export class HelmTasks {
    * Returns list of tasks which perform preflight platform checks.
    */
   startTasks(flags: any, command: Command): Listr {
-    command.warn('You can also use features rich \'OLM\' installer to deploy Eclipse Che.')
+    if (isStableVersion(flags)) {
+      command.warn('Consider using the more reliable \'OLM\' installer when deploying a stable release of Eclipse Che (--installer=olm).')
```
Comment (mmorhun) [inline src/tasks/installers/helm.ts:38]: I don't see why OLM is
 > more reliable

Comment (tolusha) [reply src/tasks/installers/helm.ts:38]: - no cluster admin rights
- OpenShift manages installation

Comment (mmorhun) [reply src/tasks/installers/helm.ts:38]: Maybe then `easier to use`?

Comment (tolusha) [reply src/tasks/installers/helm.ts:38]: Not sure, see the related issue.

**Pull request merged**